webapp/WEB-INF/conf/plugins/forms-documentproducer.properties

```diff
@@ -65,5 +65,9 @@ forms.pdfgenerate.image.fitWidth=100f
 forms.pdfgenerate.image.fitHeight=100f
 forms.pdfgenerate.image.align=0
 
+#PROPERTIES ENTRYTYPE
+forms.pdfgenerate.entrytype.geolocalized.id=109
+forms.pdfgenerate.entrytype.field.title.adress=adress
```

Comment:
address is misspelled. Where are these properties documented ? When should I use them, what values should I put ?